src/main/java/com/gizasystems/automationexercise/pages/CartPage.java

```diff
@@ -7,11 +7,10 @@
 public class CartPage {
     // Variables
     private SHAFT.GUI.WebDriver driver;
-
     // Locators
     private final By proceedToCheckout_btn = By.cssSelector(".btn.btn-default.check_out");
     private final By productName_h4 = By.xpath("//td[@class='cart_description']//h4");
-    private final By viewCart_a = By.xpath("//div[contains(@class,'confirm')]//a[@href='/view_cart']");
+    private final By viewCart_a = By.cssSelector("#header > div > div > div > div.col-sm-8 > div > ul > li:nth-child(3) > a");
```

Comment:
This is not a relative path, why did you change it ? :D

Reply:
It only passed with this cssSelector

Reply:
I don't think so :D. Please share the error and let us know to handle it better. We will not be using absolute paths :D